docs/api/static-transfers.md

```diff
@@ -0,0 +1,79 @@
+# Static Transfer API
+
+The Static transfer API is a non-interactive API for downloading LFS objects.
+Its main intent is to provide simple interoperability with existing data
+storage solutions by offloading serving LFS objects to them while using
+git and git LFS for version management.
+
+Support for it by Git LFS clients and servers is optional.
+
+Git LFS clients MAY implement this transfer API in a fallback manner:
+first attempting to download the LFS object using the Static transfer
+API and if this fails (e.g., the manifest file does not exist,
+oid cannot be found in the manifest file) use the Basic transfer API.
+
+## Downloads
+
+Downloading an object first requires fetching a manifest file from using
+HTTP from the remote server. The proposed location of the manifest file
+is `https://<remote domain>/.well-known/git-lfs-manifest.json`. The
+location is configured using the `lfs.staticurl` configuration option,
+or remote-specific `remote.{name}.lfsstaticurl` configuration option.
+Both of them can be provided in `.lfsconfig` file as well.
+
+The manifest file MUST contain a JSON payload that looks like this:
+
+```json
+{
+  "version": "1",
+  "transfer": "static",
+  "objects": [
+    {
+      "oid": "1111111",
+      "size": 123,
+      "authenticated": true,
+      "actions": {
+        "download": {
+          "href": "https://some-download.com/1111111",
+          "header": {
+            "Authorization": "Basic ..."
+          },
+          "expires_at": "2016-11-10T15:29:07Z"
```

Comment:
I don't think we really want an `expires_at` component here.  If this is supposed to be static, let's make people provide persistent, cacheable URLs.

Reply:
I was thinking here about allowing to cache discrepancies between manifest caching headers and individual entry token expiration. So I could have a long lasting (a year) token here and I can specify precise token expiration here. Then my caching of the manifest file might be some other value, just because a simple implementation using a regular HTTP server would look at a manifest file like static file. So I wanted to allow the client to detect that maybe file caching is not in sync with token expiration. And retry the manifest again. Because ideally that file got replaced in that year.

But sure, I can remove those tokens as well. I would leave `header` in though. But we can remove expiration.